src/System.Runtime.InteropServices.WindowsRuntime/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcore50aot-Windows_NT;
+      netstandard1.3-Windows_NT;
```

Comment:
I didn't think about the build all traversal, but the point of this was that non-Windows will use the `netstandard1.3`. I think that even when we enable the build all traversal, it won't build both, as depending on which vertical you are building for, netstandard1.3-Windows_NT will always be prefered over netstandard1.3, is that not the case?